test_views.py

```
@@ -0,0 +1,315 @@
+            return JsonResponse({'error': 'amount must be a positive number'}, status=400)
+        
+        order = Order.objects.create(
+            user_id=user_id,
+            total_amount=amount,
+            status=status
+        )
+        logger.info(f"Order created: {order.id}")
+        return JsonResponse({'order_id': order.id})
+    except json.JSONDecodeError:
+        return JsonResponse({'error': 'Invalid JSON'}, status=400)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)